zerver/openapi/zulip.yaml

```diff
@@ -1382,37 +1382,34 @@ paths:
                                     - delete
                                 streams:
                                   type: array
+                                  deprecated: true
                                   description: |
-                                    Array of objects, each containing
-                                    details about a channel that was deleted.
+                                    Array of objects, each containing stream_id of the channel
+                                    that was deleted.
+
+                                    **Deprecated** and will be removed in a future release.
```

Comment:
There should be "Changes" entry for all the API changes and the corresponding entries should be done in `changelog.md` as well.

Reply:
The latest feature level in `changelog.md` is 323, so these changes will be for feature level 324. Point 11 in https://zulip.readthedocs.io/en/latest/documentation/api.html#step-by-step-guide mentions the steps needed to be done while changing API.